src/Common/SymbolIndex.h

```diff
@@ -43,6 +45,9 @@ class SymbolIndex : private boost::noncopyable
     const Symbol * findSymbol(const void * address) const;
     const Object * findObject(const void * address) const;
 
+    /// Retrieves the clickhouse binary object.
+    inline const Object * getSelf() const { return findObject(&dummy_var); }
```

Comment:
Add comment why this works.

Reply:
It's quite obvious that static variables are located inside binary/DSO that contains translation unit where such variable is declared.